export const create = async () => {
  try {
  } catch (error) {
    throw new Error('FS operation failed');
  }
};

create();
